java/squeek/appleskin/helpers/DynamicFood.java

```diff
@@ -0,0 +1,21 @@
+package squeek.appleskin.helpers;
+
+import net.minecraft.item.ItemStack;
+
+/**
+ * An interface for foods whose hunger/saturation restoration can vary by the ItemStack.
+ */
+public interface DynamicFood
+{
+	/**
+	 * @param stack the stack to check
+	 * @return how much hunger an instance of this food will restore in total, including base restoration.
+	 */
+	int getDynamicHunger(ItemStack stack);
```

Comment:
These should take a PlayerEntity parameter too, for things like The Spice of Life that modify food values on a per-player basis.

Reply:
Ah yeah, I guess the calls of this code should be moved to the PlayerEntity-accepting section of the FoodHelper.